crypto/ec/ec_lcl.h

```diff
@@ -154,6 +154,10 @@ struct ec_method_st {
     int (*field_sqr) (const EC_GROUP *, BIGNUM *r, const BIGNUM *a, BN_CTX *);
     int (*field_div) (const EC_GROUP *, BIGNUM *r, const BIGNUM *a,
                       const BIGNUM *b, BN_CTX *);
+    /* computes the multipicative inverse of a in the field, storing the result in r.
```

Comment:
Multi-line comment style. First line should be "/*" on its own.

Reply:
fixed! thanks!